<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDriver</class>
 <widget class="QWidget" name="AddDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>329</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	background-color: #0089D9;
	border: none;
	color: white;
	border-radius: 7px;
}

QPushButton:pressed{
	background: #00598C;}</string>
   </property>
   <property name="text">
    <string>Добавить водителя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMiddlename">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>293</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLicencenumber">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>413</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>173</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;
</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditBirthday">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>353</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>233</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-width: 1px;
    border-style: solid;
    border-color: black;
	border-radius: 7px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер водительского удостоверения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата рождения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
